src/tutorials/zika.rst

```diff
@@ -0,0 +1,300 @@
+==================================
+Detailed analysis - Zika evolution
+==================================
+
+This tutorial dissects the :term:`single-build workflow<workflow>` used in the quickstart tutorial. We will first make the build step-by-step. Then we will automate this stepwise process in a :term:`workflow`.
+
+.. note::
+
+   The difference between a :term:`workflow` and :term:`build` isn't obvious with single-build workflows such as this example Zika workflow, but will become more distinct in multi-build workflows such as the SARS-CoV-2 workflow.
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+Prerequisites
+=============
+
+1. :doc:`Install Nextstrain </install>`.
+2. Run through the :doc:`quickstart<quickstart>`. This will verify your installation.
+
+Setup
+=====
+
+1. Activate the ``nextstrain`` conda environment.
+
+   .. code-block:: bash
+
+      conda activate nextstrain
+
+2. Change directory to the Zika :term:`workflow repository` downloaded in the quickstart.
+
+   .. code-block:: bash
+
+      cd zika-tutorial
+
+3. Create a folder for results.
+
+   .. code-block:: bash
+
+      mkdir -p results/
+
+4. Additionally, if you installed Nextstrain with the :term:`Docker runtime<runtime>`, start Docker and enter the runtime.
+
+   .. code-block:: bash
+
+      nextstrain shell .
+
+   .. note::
+
+      The dot (``.``) as the last argument indicates that your current directory (``zika-tutorial/``) is the working directory. Your command prompt will change to indicate you are in the Docker runtime. If you want to leave the runtime, run the command ``exit``.
+
+Run a Nextstrain Build
+======================
+
+:term:`Nextstrain builds<build>` typically require the following steps:
+
+.. contents::
+   :local:
+
+Prepare the Sequences
+---------------------
+
+.. admonition:: Sequence data overview
+   :class: dropdown, note
+
+   A :term:`Nextstrain build<build>` typically starts with a collection of pathogen sequences in a single `FASTA <https://en.wikipedia.org/wiki/FASTA_format>`_ file and a corresponding table of metadata describing those sequences in a tab-delimited text file. For this tutorial, we will use example data containing 34 virus sequences.
```

Comment:
Since this is a tutorial, I don't think we should hide away this contextual material on sequence data in a disclosable element. It seems crucial to understanding the data used by Nextstrain and the data requirements.

Reply:
The first time I went through this, this explanatory text was so long that I got lost where in the sequential process I was. I think the new table of contents improves that experience, so I've reverted the change here: 527d1efbd1d2dd57cae5f1f8cc386c3d6da49190